web-console/src/views/load-data-view/load-data-view.tsx

```diff
@@ -408,11 +416,37 @@ export class LoadDataView extends React.PureComponent<LoadDataViewProps, LoadDat
   };
 
   render() {
-    const { step } = this.state;
+    const { step, continueToSpec } = this.state;
+    if (!continueToSpec) {
+      return (
+        <div className={classNames('load-data-continue-view')}>
```

Comment:
everything returned in the `load-data-view` should have that class on it, you should set the class to `load-data-view continue-view` and then in the scss do:

```scss
.load-data-view {
  &.continue-view {
    ...
  }
}
```